pynq/lib/arduino/arduino_grove_gesture.py

```diff
@@ -0,0 +1,100 @@
+
+from . import Arduino
+from . import ARDUINO_GROVE_I2C
+from enum import Enum

```

Comment:
As a pep8 rule, built-in module should be imported first. So line 4 should come first.